packages/grid/theme/lumo/vaadin-grid.js

```diff
@@ -1,9 +1,2 @@
-import '@vaadin/vaadin-lumo-styles/color.js';
-import '@vaadin/vaadin-lumo-styles/font-icons.js';
-import '@vaadin/vaadin-lumo-styles/sizing.js';
-import '@vaadin/vaadin-lumo-styles/spacing.js';
-import '@vaadin/vaadin-lumo-styles/style.js';
-import '@vaadin/vaadin-lumo-styles/typography.js';
```

Comment:
These imports are already present in `vaadin-grid-styles.js` file:

https://github.com/vaadin/web-components/blob/cc16ae365e44ac970a6bc9fa6c189172e5f4115d/packages/grid/theme/lumo/vaadin-grid-styles.js#L1-L6